### DISCUSSION TITLE
Are there templates for Codemagic CI/CD or has anyone set this up ?

### DISCUSSION THREAD

**Discussion by stephenhauck** [unoplatform/uno#15290] [General]
I don't build locally because it's error prone.
I use Codemagic for my Flutter and they have a MAUI section .. 
Has anyone used any Ci / CD  they can share information about ?

FWIW .. I would pay for  a template for this that 

- Pulls from the GitHub repo
- Bulds iOS 
- Buolds Android
- Submits to Apple store 
- Submits to Google play ...

**Comment by jeromelaban** [comment]
We do not yet have documentation from Uno that uses CodeMagic, but since Uno Platform apps are the same as .NET Android/iOS Apps, the MAUI docs apply in the same way. Building for Wasm is similar to Blazor, and building for Skia desktop targets is the same as publishing a standard .NET desktop app.
